### ISSUE TITLE
Hardcoded paths prevent parallel execution

### ISSUE THREAD

**Issue by aozalevsky** [salilab/IHMValidation#44]
It would be nice to have the ability to update the PDB-Dev in parallel. Together with the recent updates in #38 it would allow to rebuild the whole repo (with recalculated values) in under 2 minutes on a modern 32-128 core node. 

So far I identified several places which interfere with parallel execution:

https://github.com/salilab/IHMValidation/blob/f1aec61bc2f8de7717f20024b7fb712d94733d4f/master/pyext/src/validation/__init__.py#L737-L739

uses a hardcoded `test.cif` as a temporary filename

https://github.com/salilab/IHMValidation/blob/f1aec61bc2f8de7717f20024b7fb712d94733d4f/master/pyext/src/validation/utility.py#L475-L490

removes any temp files by mask, including temp files generated for other structures. It specifically hits sascif processing (looks like other files are not reread again. at least when molprobity and excluded volume are already recalculated).

https://github.com/salilab/IHMValidation/blob/f1aec61bc2f8de7717f20024b7fb712d94733d4f/master/pyext/src/validation/sas.py#L86
https://github.com/salilab/IHMValidation/blob/f1aec61bc2f8de7717f20024b7fb712d94733d4f/master/pyext/src/validation/sas.py#L134
https://github.com/salilab/IHMValidation/blob/f1aec61bc2f8de7717f20024b7fb712d94733d4f/master/pyext/src/validation/sas.py#L370-L372
https://github.com/salilab/IHMValidation/blob/f1aec61bc2f8de7717f20024b7fb712d94733d4f/master/pyext/src/validation/sas.py#L379

https://github.com/salilab/IHMValidation/blob/f1aec61bc2f8de7717f20024b7fb712d94733d4f/master/pyext/src/validation/sas.py#L381-L382

temp files for sas processing.

**Comment by aozalevsky** [comment]
Fully addressed in v2.0. Keeping  open until the merge.
